.github/workflows/dotnet-build-and-release.yml

```diff
@@ -15,6 +15,11 @@ on:
         description: The relative directory inside the repo where the build artifacts to publish for release will be located
         required: false # leave empty for non-release build
         type: string
+    secrets:
+      token:
+        description: 'Secret token from caller workflow to access private repo'

```

Comment:
The description says "access private repo" but we are trying to grant access to private packages

Reply:
changed description repo -> packages